types/unidecode/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for unidecode 0.1
+// Project: https://github.com/FGRibreau/node-unidecode
+//          http://blog.fgribreau.com/2012/05/unidecode-for-javascript-nodejs.html
```

Comment:
This link seems irrelevant, it should be the project url, rather than a blog url